lib/widgets/graph_component.dart

```diff
@@ -1,212 +1,216 @@
-// Copyright (c) 2023- All flutter_graph_view authors. All rights reserved.
-//
-// This source code is licensed under Apache 2.0 License.
-
-import 'dart:async';
-
-import 'package:flame/events.dart';
-import 'package:flame/game.dart';
-import 'package:flutter/material.dart';
-import 'package:flutter_graph_view/flutter_graph_view.dart';
-
-/// Boot of graph.
-///
-/// 图构建器
-class GraphComponent extends FlameGame
-    with
-        PanDetector,

```

Comment:
Missing PanDetector, onPanUpdate cannot form a valid callback.
Before: 

https://github.com/graph-cn/flutter_graph_view/assets/15630211/dac9288c-c882-43ee-949f-00faa2a88cd1

The PC end does not work properly Did I handle something wrong.
Now: 

https://github.com/graph-cn/flutter_graph_view/assets/15630211/7a2d9c2a-bb17-4b9a-ad97-9e643e5e8b0a

Reply:
@CorvusYe yeap, you need to add handling of Pan gestures using the ScaleDetector.
I have done this just now. Please, check out the new commit.
https://github.com/graph-cn/flutter_graph_view/pull/12/commits/c77964e204c19d5c9cd9b46c69b352fa233d21ce

Also, you need to call `super.onScaleUpdate` in case if you need to override this in the graph_component implementation